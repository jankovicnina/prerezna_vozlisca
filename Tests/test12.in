306 1427
85 93 7.2
80 62 3.92
244 276 6.68
169 190 7.36
36 297 4.11
62 290 1.93
47 170 5.75
11 233 1.28
35 28 0.49
141 151 3.32
81 158 4.02
74 0 4.41
275 106 0.64
192 57 3.39
286 94 7.34
166 23 9.37
235 169 5.49
16 260 -0.79
183 214 2.82
200 40 2.29
43 58 9.95
142 234 5.67
38 121 7.31
121 45 7.41
198 59 3.08
52 215 5.78
36 291 9.54
267 212 0.86
135 284 6.23
199 90 -1.12
106 65 8.86
283 288 3.01
61 48 -0.3
106 36 5.82
222 142 3.9
221 71 2.75
181 300 9.58
128 138 8.89
265 267 6.37
137 256 1.08
305 96 9.26
276 39 5.9
273 287 9.51
182 63 3.86
70 263 1.86
196 117 1.01
228 147 3.51
70 195 1.24
266 26 7.3
80 183 3.42
8 275 1.94
15 200 3.77
242 200 7.93
189 262 -1.29
103 257 1.47
258 140 0.71
213 267 0.61
37 123 0.58
117 303 3.47
305 249 2.37
124 46 -1.08
263 182 4.62
120 274 8.87
174 130 3.94
3 302 5.23
253 50 6.15
32 122 2.66
236 107 -0.24
196 8 3.75
124 95 -1.21
173 29 7.0
222 183 1.84
233 92 4.86
298 58 5.28
109 36 1.62
112 228 2.13
7 49 0.1
5 70 2.62
54 252 2.26
109 302 7.62
115 110 0.06
276 246 4.06
14 298 8.91
42 13 3.26
231 106 4.32
54 77 2.72
51 47 8.07
57 41 5.27
200 67 5.89
94 257 1.74
244 140 6.34
187 189 6.95
47 247 2.05
279 198 8.55
172 243 3.84
9 149 2.83
129 300 1.9
259 235 5.15
131 255 6.13
71 203 5.93
47 62 2.64
278 264 2.24
194 296 7.66
90 116 6.89
128 267 2.72
177 21 2.04
74 19 1.63
75 73 0.21
93 222 0.78
97 171 3.87
276 209 2.62
277 60 1.07
245 300 7.92
31 251 7.16
69 9 6.97
227 213 -0.28
11 7 4.53
165 243 1.96
243 214 4.88
83 98 7.16
28 241 3.81
260 263 4.47
185 251 4.03
264 76 2.16
6 297 2.82
187 17 2.88
148 248 6.54
32 260 9.06
96 180 6.84
156 90 3.41
134 154 0.93
266 77 7.56
279 277 6.38
162 159 -0.08
303 195 9.0
232 155 7.71
107 268 5.32
259 286 0.48
39 269 7.67
57 128 2.59
68 169 2.5
229 152 -0.57
115 138 -0.86
102 206 3.23
138 52 0.28
255 39 8.68
180 130 -0.62
68 210 7.07
232 10 3.1
5 259 -0.21
198 99 4.12
135 103 0.91
2 257 8.53
227 143 6.84
40 207 4.3
51 197 8.7
114 91 9.66
84 245 4.18
110 216 2.72
113 269 4.18
261 41 0.36
15 5 6.1
43 134 4.15
268 57 8.7
17 149 -0.6
297 59 2.84
105 213 8.36
4 134 7.48
78 187 8.62
18 57 6.55
303 156 3.96
150 267 7.05
219 103 7.97
256 241 2.43
69 297 6.45
161 292 2.77
7 178 9.22
250 110 8.41
117 151 1.98
192 197 9.81
91 207 1.72
281 159 3.48
285 228 4.43
2 22 4.89
169 35 9.94
128 23 8.13
89 23 5.76
300 152 9.21
127 168 3.13
167 103 6.26
44 112 0.3
226 117 -1.13
295 167 0.27
268 17 2.15
269 260 0.27
126 277 7.6
225 297 9.02
269 32 -0.37
112 282 5.12
244 8 -0.99
284 93 2.12
135 22 7.18
301 188 7.89
188 110 9.78
28 240 -0.05
51 301 7.92
114 124 9.91
74 88 -1.82
164 26 5.61
172 175 4.67
2 145 4.95
157 125 6.91
151 41 9.0
21 197 3.75
225 177 8.58
254 1 7.33
47 28 4.8
32 7 2.99
66 171 5.1
114 15 0.85
202 250 0.18
234 156 1.74
41 191 0.49
55 216 6.71
138 240 0.96
237 126 2.83
305 183 2.79
164 204 -1.96
31 299 4.84
169 116 7.59
117 234 0.23
7 29 7.99
264 12 3.32
220 112 3.1
100 258 5.9
127 226 6.56
292 213 4.63
198 86 9.21
32 35 7.18
62 146 6.22
209 141 2.47
203 263 0.63
114 228 2.83
29 221 3.52
96 26 2.21
4 56 8.57
174 11 1.79
302 20 0.3
89 132 3.67
178 27 4.75
3 89 0.1
217 127 6.71
286 118 4.58
182 114 4.64
123 229 3.56
284 289 9.29
147 214 2.57
12 168 2.7
105 275 7.78
202 289 3.74
290 24 3.55
241 91 9.98
268 250 4.49
75 125 9.04
241 49 2.39
134 82 1.63
30 137 8.67
169 264 7.0
51 152 7.76
116 204 5.11
125 248 6.36
213 301 0.06
195 22 3.07
242 195 1.92
33 214 9.91
282 291 7.15
57 278 0.97
20 295 3.33
230 11 8.29
305 231 5.59
185 110 0.9
45 239 7.25
154 79 5.1
151 152 0.63
124 137 6.86
236 48 -1.84
103 295 9.33
132 281 8.65
189 289 2.45
9 200 -0.81
230 295 9.04
264 158 7.71
60 22 7.47
57 230 0.64
302 110 1.59
85 155 8.63
193 220 7.33
276 97 7.38
143 168 6.69
156 201 9.96
84 184 4.9
99 205 6.21
161 202 7.79
161 18 7.25
297 303 6.54
1 238 -0.23
193 187 1.21
246 65 1.79
77 32 2.39
292 202 -0.13
85 27 9.54
235 258 4.21
14 292 5.15
150 48 6.66
149 299 9.7
229 181 1.96
92 78 1.27
188 254 1.37
150 265 0.96
190 47 -1.77
219 24 7.2
139 189 1.16
86 188 0.23
88 22 9.49
58 149 5.27
243 219 7.55
132 189 4.03
235 39 6.34
0 22 7.98
276 29 3.15
118 290 -1.09
30 289 5.27
40 267 5.96
239 97 1.93
60 176 4.71
304 83 1.57
89 118 8.56
86 154 1.92
206 165 -0.59
110 288 0.06
15 115 6.94
111 246 7.43
140 180 8.27
182 75 4.84
76 272 9.02
76 155 -0.85
216 110 4.45
70 102 9.31
77 20 2.25
67 236 9.37
135 190 2.59
276 162 3.71
58 164 7.56
188 76 5.61
97 183 0.92
24 94 0.83
54 143 1.65
181 74 4.03
180 46 5.87
92 109 6.65
172 251 7.35
250 197 -0.1
224 117 1.58
143 32 8.57
44 81 6.03
87 108 1.15
140 189 5.76
170 20 2.65
13 198 8.09
240 66 3.06
80 68 8.73
168 165 8.22
191 258 3.94
45 217 1.62
87 258 9.66
33 43 -0.69
292 16 3.4
33 124 3.16
303 63 6.89
124 180 6.22
157 259 7.86
2 194 3.38
290 78 2.85
175 163 -0.29
4 241 0.67
263 39 2.07
159 274 4.91
59 111 0.1
73 2 1.88
16 89 8.97
182 245 6.27
48 218 3.12
114 174 0.02
168 153 8.99
12 105 8.99
189 304 4.07
117 218 1.23
247 253 0.23
248 232 0.06
206 296 7.77
22 5 5.42
80 271 -0.21
180 111 8.5
79 88 7.66
85 113 3.8
158 294 5.38
28 93 8.43
42 84 7.02
212 39 3.72
132 60 3.65
64 116 7.73
246 173 2.42
266 189 9.74
260 54 4.27
222 289 9.29
49 220 7.72
253 194 6.65
260 45 3.3
274 303 0.26
35 212 9.71
249 225 6.43
78 137 9.14
93 204 9.27
205 172 6.08
78 266 5.05
17 82 8.37
17 81 -0.0
25 24 1.93
102 262 6.95
155 19 2.17
101 242 -1.36
158 196 9.56
54 6 7.32
208 51 8.4
219 291 7.97
94 117 5.83
94 195 5.54
102 231 9.08
81 259 2.84
27 62 7.67
272 216 6.38
18 267 3.76
45 263 4.81
1 85 9.2
53 74 7.56
161 120 4.21
153 110 7.08
133 26 1.67
301 36 -0.39
4 97 4.02
287 248 -0.11
225 116 2.96
244 56 2.37
86 144 1.26
217 212 4.24
193 173 7.68
213 121 4.09
31 198 3.11
180 58 3.38
111 295 4.65
165 157 2.88
171 27 7.5
139 68 7.29
162 234 8.93
234 244 2.93
225 226 0.8
138 266 3.09
143 174 9.46
162 97 1.21
151 64 8.99
113 59 1.08
184 161 8.48
260 225 4.47
233 301 1.99
143 226 -0.1
256 80 2.62
79 8 2.33
174 299 5.92
104 242 6.42
39 137 7.44
29 55 5.68
235 1 0.84
269 216 1.79
122 70 2.18
285 90 7.42
272 240 9.54
109 151 -0.78
27 150 3.98
214 40 8.79
24 179 8.84
83 34 9.16
147 3 3.75
97 44 -0.72
146 253 0.26
40 117 4.11
24 14 6.95
187 221 8.48
210 116 1.16
18 59 5.36
50 132 8.36
72 282 2.5
192 292 3.26
249 264 0.78
108 55 7.87
142 125 0.33
231 12 2.28
273 169 2.41
124 250 -0.79
187 162 3.28
293 196 1.55
288 12 9.57
174 49 5.64
147 175 -1.07
234 196 -0.83
169 134 -0.12
230 152 -0.81
10 251 1.49
294 133 9.03
118 97 5.99
261 157 5.71
81 236 0.58
90 271 6.04
45 153 7.59
159 305 3.41
104 144 5.14
278 58 7.78
23 129 -0.51
138 200 8.73
281 246 2.86
52 16 7.63
5 32 3.29
115 274 3.81
250 142 7.28
192 148 9.29
258 8 2.8
233 151 -1.19
187 237 7.83
65 87 8.76
88 298 5.6
54 84 0.84
228 163 3.62
39 167 7.43
129 259 2.41
263 272 1.54
80 69 -1.96
35 290 -1.71
141 1 0.88
249 150 1.0
281 227 5.68
185 248 2.76
224 166 7.99
41 11 8.09
197 289 3.35
147 242 0.38
104 200 5.53
51 7 0.72
85 263 4.85
74 227 0.39
190 251 -0.96
283 249 5.22
32 187 3.82
223 66 9.38
182 224 6.76
243 90 8.08
160 176 9.14
173 104 8.78
290 79 3.34
67 287 1.77
143 159 9.7
258 232 8.32
293 217 -0.61
294 263 5.57
222 32 2.63
214 258 9.67
94 124 0.56
112 138 8.58
289 294 6.65
96 169 4.78
53 209 9.58
229 87 1.83
209 2 0.76
288 283 9.01
212 56 -1.2
20 113 9.37
117 103 8.95
149 80 6.49
98 207 8.85
161 29 9.58
41 162 1.96
226 291 9.79
265 136 7.85
243 215 -0.42
266 49 3.07
225 14 9.89
24 156 9.03
196 50 0.27
203 39 2.83
277 143 2.31
73 250 5.02
37 70 9.99
142 151 4.14
289 2 -0.72
89 271 2.65
117 115 -0.8
159 133 7.63
36 38 3.1
77 202 4.33
268 182 4.49
136 102 7.4
0 188 8.5
138 95 2.65
272 259 6.51
165 216 5.7
63 184 6.54
134 102 7.85
2 160 2.72
294 164 7.98
208 24 0.6
133 39 5.88
296 262 8.55
257 16 2.32
282 189 3.6
77 83 -1.5
14 130 4.87
19 182 5.83
264 10 7.71
122 114 3.67
21 16 1.28
41 139 6.25
280 218 8.37
37 147 2.27
139 203 -1.19
197 22 6.8
41 146 6.27
274 109 4.71
278 136 2.27
157 51 4.44
191 276 1.1
116 174 2.85
7 176 0.81
127 100 0.9
121 206 -1.27
168 106 2.31
305 45 4.6
303 14 2.36
100 300 1.75
127 34 8.21
101 274 1.85
261 118 2.22
17 185 2.44
286 132 5.67
228 73 7.46
62 125 6.04
88 132 -1.24
270 169 4.45
290 233 8.75
36 121 0.42
12 56 6.82
268 258 0.85
298 276 -1.01
16 287 1.74
264 98 -0.17
181 253 0.44
207 169 2.61
61 297 2.59
217 185 6.03
214 240 -0.14
185 124 8.63
32 276 7.34
148 258 0.52
131 153 8.49
61 24 -0.87
44 177 -0.95
78 264 9.95
173 156 7.08
202 54 9.51
263 27 8.24
156 179 5.96
67 194 5.04
7 191 7.09
148 168 4.03
61 223 2.05
262 55 -0.7
203 190 9.7
232 86 9.61
52 302 0.02
102 149 1.11
74 207 1.24
90 217 7.09
183 84 6.86
24 126 6.44
146 271 -0.06
73 11 9.9
126 87 6.85
234 47 8.62
237 249 4.37
197 264 0.35
65 146 1.66
242 3 8.26
21 122 0.8
151 229 6.0
180 235 7.36
82 207 1.58
19 293 0.84
142 225 8.89
139 97 1.61
170 209 8.43
17 0 8.81
2 0 -0.21
48 30 6.83
120 203 5.83
155 143 2.41
138 211 4.71
226 154 5.3
293 268 0.14
179 305 9.58
258 25 1.61
11 262 7.13
275 114 8.3
147 45 2.43
223 284 8.04
42 156 5.27
235 228 3.4
218 216 0.66
276 256 1.17
185 265 8.77
54 192 6.99
204 181 -0.88
167 181 6.2
55 174 0.57
108 2 6.63
120 225 2.13
91 186 -0.66
10 1 3.91
167 254 7.4
140 110 9.61
147 132 5.38
52 31 7.1
258 144 8.3
277 228 9.22
57 185 5.46
292 216 0.89
291 126 8.68
7 135 0.74
74 73 7.56
111 258 6.85
57 304 4.48
128 283 0.21
249 168 8.73
35 243 6.38
89 103 3.98
147 72 1.31
10 111 4.41
125 175 1.02
284 98 0.21
215 106 1.29
262 12 4.93
288 216 4.23
2 75 4.91
35 87 9.5
174 203 -1.17
257 187 6.17
81 280 2.36
254 74 5.91
80 196 5.72
157 216 -1.14
82 127 9.07
283 264 5.47
149 63 0.66
100 106 1.14
295 87 5.05
77 16 7.38
6 54 5.3
301 239 6.12
19 137 7.35
219 185 7.51
179 175 4.53
235 237 8.09
305 17 -1.15
287 44 9.96
30 159 0.4
254 121 -1.26
33 145 7.34
33 244 0.8
285 193 6.65
238 90 -0.37
114 38 -0.45
229 147 1.53
128 72 -0.84
136 101 7.13
149 301 6.69
248 47 2.25
7 102 7.38
224 155 7.0
141 26 8.6
235 262 8.03
187 41 5.92
55 111 2.92
114 140 7.46
242 303 4.22
1 241 5.2
41 184 5.07
16 205 4.26
131 76 3.75
283 179 9.44
104 162 9.37
22 13 7.59
257 176 4.46
26 186 3.86
279 24 0.61
132 193 9.15
192 269 8.8
215 3 2.94
170 187 6.83
142 148 6.13
113 55 5.1
276 91 0.53
103 168 5.41
16 232 1.87
24 223 -1.07
60 224 1.61
31 14 4.78
27 23 1.26
148 263 1.44
34 19 -0.71
254 119 5.82
97 100 6.92
223 277 1.33
34 242 9.25
164 64 8.79
157 80 6.24
40 121 5.56
298 199 6.79
153 220 6.65
221 191 9.91
58 210 9.93
256 0 5.4
218 99 6.93
99 291 9.06
293 19 8.91
286 241 0.17
48 27 8.74
252 49 -0.77
127 61 -0.08
56 42 -0.76
178 75 4.43
6 214 3.11
51 104 9.78
193 201 -0.06
50 237 6.92
123 279 4.44
291 269 3.68
203 182 7.35
160 66 -0.24
277 262 4.7
128 297 4.67
226 209 1.82
282 222 2.62
282 160 9.62
305 198 -0.52
298 281 3.75
11 125 2.02
7 10 2.13
250 72 2.73
228 191 1.8
157 180 5.36
155 288 0.46
124 202 5.38
258 200 0.61
174 34 -0.97
268 295 0.08
112 118 0.42
292 36 5.67
91 128 5.85
291 58 6.75
142 104 2.09
6 144 0.73
108 120 9.57
203 159 0.6
199 10 -0.1
297 183 0.75
70 110 3.08
52 171 4.93
188 220 2.77
87 51 2.97
96 294 6.32
272 302 8.06
158 25 2.86
134 85 8.37
106 273 6.08
241 71 2.15
294 196 0.2
224 251 6.28
20 163 0.1
136 79 5.29
233 253 4.16
247 128 8.63
54 217 3.75
39 110 0.48
144 110 2.06
147 160 -0.01
201 239 6.16
301 196 5.68
66 147 0.46
156 188 9.58
70 202 1.84
181 72 6.26
95 11 8.81
17 3 1.93
257 206 5.29
25 244 8.55
211 85 7.57
71 159 5.57
202 122 2.43
269 115 9.82
236 165 3.54
282 183 6.46
273 153 6.08
173 242 0.46
113 297 2.14
84 270 -1.04
197 230 9.45
75 288 2.96
63 44 2.59
198 107 -0.44
240 25 0.42
234 6 0.93
124 185 9.96
217 200 2.98
59 109 1.42
109 142 6.76
63 156 1.44
81 86 -0.07
268 64 7.47
140 305 3.31
275 61 8.53
270 280 3.61
216 88 8.34
23 12 1.23
232 15 8.12
247 212 2.73
301 176 8.46
283 246 0.05
243 58 4.89
214 53 1.44
67 55 -0.11
262 48 1.31
275 192 4.89
3 205 2.06
218 223 4.91
197 281 1.83
213 60 5.65
206 224 6.43
243 266 3.28
192 283 7.03
112 95 8.77
166 124 3.08
207 7 7.29
31 52 5.97
179 238 -0.67
40 232 -1.98
151 35 3.03
284 62 0.59
102 257 2.63
232 93 5.38
281 140 2.29
283 193 -0.06
199 279 1.07
118 199 1.33
105 95 5.67
257 294 9.18
64 4 7.46
219 303 4.58
279 60 -0.86
106 96 9.15
293 119 -0.59
165 246 7.81
140 132 5.18
45 126 4.25
289 290 4.33
25 47 6.08
281 241 3.54
154 158 4.54
57 276 8.27
135 134 0.87
219 109 2.94
112 235 6.7
145 301 3.53
245 55 -0.34
251 284 -0.95
239 53 -0.28
270 127 4.0
59 40 7.68
150 243 5.93
93 203 7.99
239 132 5.81
152 104 1.39
34 33 3.78
205 26 6.2
168 4 6.62
172 23 4.61
136 23 6.66
57 206 1.89
233 86 -0.06
268 277 7.74
53 65 0.36
181 245 5.49
210 266 2.32
146 188 4.05
140 124 4.59
295 16 1.4
42 43 4.55
104 261 2.77
299 273 4.93
72 230 1.27
256 37 4.26
295 289 2.27
84 79 2.93
228 269 3.66
196 293 6.35
30 304 4.89
125 60 4.22
296 211 8.0
30 106 2.94
0 115 -1.16
57 212 8.69
220 107 -0.18
23 304 -0.64
110 252 -0.45
28 221 4.81
296 125 6.61
256 199 8.39
170 248 6.48
70 82 5.2
101 191 2.86
109 67 3.34
3 170 7.55
113 119 1.65
92 250 9.4
116 244 7.99
289 12 4.68
246 12 4.9
42 291 5.22
167 247 0.35
92 151 1.63
234 116 2.23
53 197 -0.66
239 277 5.65
229 8 9.58
182 210 3.58
219 292 -1.02
248 223 0.73
64 283 -1.7
177 148 6.29
261 172 5.5
56 220 -1.01
111 117 -0.04
67 16 4.1
7 154 2.54
223 60 0.19
110 175 9.79
303 87 9.77
42 27 7.37
60 147 5.31
298 193 4.37
77 48 8.4
219 137 4.58
153 212 2.82
134 70 3.78
134 243 9.46
136 84 0.24
222 5 6.09
247 278 8.21
37 218 8.35
39 138 9.67
268 48 9.61
21 270 8.66
84 191 7.27
201 14 4.02
168 226 0.81
149 139 8.55
89 275 9.45
8 155 1.41
253 20 0.18
263 106 7.49
238 167 3.68
207 90 1.8
138 196 8.98
285 286 2.24
190 258 0.11
97 269 7.7
247 255 6.34
174 7 0.11
141 157 7.64
191 255 -0.34
238 199 9.14
2 267 0.06
138 55 2.5
33 248 5.44
55 80 5.77
39 87 3.34
231 259 6.37
28 94 4.1
273 267 0.69
252 236 3.67
29 286 -0.11
9 124 5.4
131 206 5.77
0 272 6.72
121 185 3.61
113 60 3.24
198 207 1.6
303 268 9.25
122 268 3.14
55 261 3.35
256 250 6.64
146 43 0.55
62 50 1.73
199 241 8.61
215 11 3.81
241 31 1.18
170 129 0.72
198 127 -0.56
28 189 6.05
81 281 3.46
200 282 5.57
37 97 0.48
179 270 3.28
269 33 9.59
101 103 7.4
244 241 7.11
232 68 5.42
171 233 2.8
8 97 4.25
277 40 9.63
200 162 -0.49
42 36 2.31
128 288 4.27
243 116 5.85
59 107 6.35
125 4 2.11
188 248 0.14
116 112 3.81
253 192 3.02
160 142 9.07
148 110 0.3
224 248 1.79
90 74 1.85
85 228 9.42
72 202 7.46
24 87 4.35
65 123 9.12
285 186 7.12
109 113 6.47
131 274 9.45
268 206 3.32
266 249 3.94
214 42 8.3
129 206 2.65
45 178 3.12
54 15 1.11
155 165 1.63
34 97 9.99
171 166 1.76
108 219 4.87
201 64 9.92
3 44 3.47
263 101 0.69
292 228 9.81
202 7 0.76
300 4 0.55
117 19 9.64
190 145 3.19
272 122 1.96
217 113 8.58
252 195 8.66
64 97 7.42
13 195 2.49
21 88 8.2
293 204 2.55
241 174 0.15
190 72 8.85
165 158 7.72
122 187 2.71
74 156 0.58
205 73 6.78
163 18 7.09
203 239 8.31
203 291 0.45
29 155 2.09
240 43 1.3
43 144 7.88
76 183 9.99
190 173 0.72
206 0 5.85
39 85 9.86
175 182 3.97
143 215 9.59
60 240 6.8
224 15 4.01
179 299 -0.59
89 207 8.07
278 153 3.25
35 101 7.7
160 251 5.63
169 222 4.86
235 215 7.03
56 103 4.35
203 152 4.0
4 81 -0.55
25 228 2.51
261 135 4.16
38 134 6.2
133 62 6.71
144 178 7.97
76 17 5.64
280 266 5.14
212 228 5.05
2 263 8.84
1 65 5.61
65 285 9.4
165 113 1.62
224 228 0.81
285 76 7.23
282 10 8.09
215 40 3.73
218 260 7.01
279 50 8.97
190 277 1.21
160 297 8.88
87 217 2.16
41 76 8.34
298 240 1.29
40 217 9.54
85 44 0.09
152 89 3.35
77 233 1.93
297 193 3.78
26 71 3.79
166 20 0.33
162 164 5.69
90 159 5.51
145 166 -0.49
190 49 -0.36
10 70 2.75
300 227 -0.17
281 117 9.77
54 67 8.94
247 181 -1.26
273 250 4.68
281 197 1.64
245 169 7.47
51 190 9.6
108 176 5.18
14 111 9.85
199 33 1.2
19 134 2.99
268 140 2.0
259 193 5.91
160 113 1.0
162 285 7.78
25 262 1.74
252 281 9.26
256 196 5.67
295 244 4.37
230 78 8.56
43 246 2.76
271 190 -0.37
202 32 6.53
28 149 9.34
130 31 2.96
163 222 3.16
276 180 3.71
7 266 5.66
273 117 9.77
100 41 1.97
222 95 -0.34
125 8 0.0
209 38 -0.14
19 163 2.79
298 207 5.94
216 21 1.02
248 81 -1.63
5 9 6.01
65 243 3.55
184 102 0.18
6 0 3.9
66 173 1.93
280 249 0.94
195 16 3.5
11 142 0.66
0 220 4.24
274 235 -0.47
168 274 3.45
129 291 3.67
255 67 0.11
80 3 9.69
196 192 9.12
76 227 4.21
188 120 7.52
201 57 4.87
119 247 3.35
287 60 4.11
195 46 3.5
49 97 7.48
176 270 0.2
304 40 8.5
12 88 9.48
254 286 -1.82
163 17 -0.64
236 113 2.14
113 121 4.39
63 140 8.33
230 41 7.52
279 241 4.56
174 248 3.62
206 57 7.54
46 231 4.97
189 205 7.02
271 101 1.88
274 290 9.01
117 15 4.59
63 205 4.26
176 8 3.56
11 225 -1.07
49 173 9.68
98 110 -0.59
134 69 8.58
59 7 8.2
152 136 5.61
73 245 6.8
166 92 9.23
203 238 2.64
151 226 1.23
0 139 1.65
270 140 6.17
24 205 8.44
149 225 8.98
186 249 3.17
31 181 3.81
254 56 2.95
165 107 4.59
68 200 2.54
169 167 9.65
56 131 8.35
181 53 -1.0
25 39 4.22
10 123 5.49
132 42 4.94
108 7 1.23
42 203 0.94
94 261 2.17
195 264 1.39
71 204 3.18
82 104 2.27
139 3 6.54
241 285 6.13
161 224 4.04
202 237 0.79
71 20 -1.11
192 305 -0.74
246 259 3.56
214 193 4.64
25 53 -1.05
14 271 6.75
157 208 -1.07
295 285 3.25
133 227 5.62
98 151 -0.38
299 252 3.26
91 35 -0.54
222 136 7.59
1 155 9.52
52 107 9.84
224 267 3.72
111 146 9.49
15 222 4.37
134 174 8.56
11 302 4.65
70 246 7.42
70 229 1.98
244 17 4.89
153 238 2.56
161 243 4.22
73 129 -0.9
118 141 4.05
180 178 5.7
99 109 1.96
241 116 8.77
171 69 4.32
43 160 3.01
254 304 8.16
42 298 2.48
13 179 8.95
232 5 1.08
38 108 3.81
122 269 5.87
104 75 -1.19
13 134 -0.27
150 193 5.95
283 59 9.2
20 157 -1.76
238 68 9.64
147 42 2.59
16 71 1.23
260 181 9.32
37 188 7.41
254 25 1.51
221 289 3.34
89 249 5.05
221 196 4.33
215 245 2.1
214 121 0.93
64 230 5.03
266 69 -1.28
133 215 1.94
183 153 7.15
25 16 4.05
168 17 4.24
294 44 -1.0
239 82 6.88
18 70 9.25
279 261 9.71
11 73 2.19
22 120 5.11
284 164 6.7
293 205 5.24
