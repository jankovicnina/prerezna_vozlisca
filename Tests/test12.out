468 480 587 757 1072 1245 1344 1715 1730 2472 2496 2950 3197 3557 3594 3789 3896 3975 4433 5604 5788 5830 6166 6240 6694 6851 7139 7966 8247 8493 8954 9040
